kurl_util/cmd/vendorflights/main_test.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/fs"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_extractPreflightSpec(t *testing.T) {
+	tests := []struct {
+		name             string
+		wantErr          bool
```

Comment:
You should have a test for the error case, if there isn't an error case, get rid of wantErr and it's not needed.

Reply:
on it